modules/headers/headers_test.go

```diff
@@ -0,0 +1,80 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package headers_test
+
+import (
+	"fmt"
+	"net/http"
+	"reflect"
+	"testing"
+
+	"github.com/philippta/flyscrape"
+	"github.com/philippta/flyscrape/modules/headers"
+	"github.com/philippta/flyscrape/modules/hook"
+	"github.com/philippta/flyscrape/modules/starturl"
+	"github.com/stretchr/testify/require"
+)
+
+func TestHeaders(t *testing.T) {
+	testCases := []struct {
+		name        string
+		headersFn   func() headers.Module
+		wantHeaders map[string][]string
+	}{
+		{
+			name: "empty custom headers",
+			headersFn: func() headers.Module {
+				return headers.Module{
+					Headers: map[string]string{},
+				}
+			},
+			wantHeaders: map[string][]string{"User-Agent": {"flyscrape/0.1"}},
+		},
+		{
+			name: "non-empty custom headers",
+			headersFn: func() headers.Module {
+				return headers.Module{
+					Headers: map[string]string{
+						"Basic":      "ZGVtbzpwQDU1dzByZA==",
```

Comment:
Although it doesn't matter for the test, but `Basic` is not an actual header. 

I think what you meant to write here is:
```go
"Authorization": "Basic ZGVtbzpwQDU1dzByZA==",
```

Otherwise great tests! 👍

Reply:
yes, that is what i meant